hawkular-agent-core/src/main/java/org/hawkular/agent/monitor/storage/NotificationDispatcher.java

```diff
@@ -63,14 +62,4 @@ public NotificationDispatcher(StorageAdapter storageAdapter, String feedId) {
                     }
                 });
     }
```

Comment:
From what I can tell the mechanism will still work.  The difference is only that we'll send a RESOURCE_ADDED notification if the resource is newly discovered OR updated in some way.  Because we already protect against duplicate RESOURCE_ADDED events we should still be OK, despite potentially sending more unnecessary notifications due to updates.  Because we can't distinguish between adds and updates, it precludes our ability to generate explicit update notifications (but note that the previous version of InventoryListener did not provide a resourcedUpdated() method, so we probably haven't lost anything in this regard).